import-export-cli/integration/apim/apim.go

```diff
@@ -853,7 +853,7 @@ func (instance *Client) DeleteSubscriptions(apiID string) {
 }
 
 // AddApplication : Add new Application to APIM
-func (instance *Client) AddApplication(t *testing.T, application *Application, username string, password string) *Application {
+func (instance *Client) AddApplication(t *testing.T, application *Application, username string, password string,doClean bool) *Application {
```

Comment:
Shall we add a space before "doClean"? (Also, change the other places where applicable)